doc/api/next_api_changes/behavior/23031-AL.rst

```diff
@@ -0,0 +1,2 @@
+The encoding of style file is now specified to be utf-8
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Because we load the style library files on import, we have not been importable with
non-utf-8 compatible locale encoding settings so there is no deprecation.  This change
formalizes the status quo.
```

Reply:
"we have not been importable with" sounds wrong.